src/components/References.vue

```diff
@@ -1,30 +1,113 @@
 <template>
   <section id="references">       
     <div class="text-content">
-      <h2 v-html="text.subtitle1" />
-      <p v-html="text.p1" />
-      <p v-html="text.p2" />
-      <p v-html="text.p3" />
+      <h2>{{ text.title }}</h2>
+      <div class="usa-accordion usa-accordion--bordered">
+        <div
+          v-for="reference in text.references"
+          :key="reference.subTitle"
+        >
+          <h3 class="usa-accordion__heading">
+            <button
+              class="usa-accordion__button"
+              aria-expanded="false"
+              :aria-controls="reference.id"
+            >
+              {{ reference.subTitle }}
+            </button>
+          </h3>
+          <div
+            :id="reference.id"
+            class="usa-accordion__content usa-prose target"
+          >
+            <p><span v-html="reference.reference" /></p>
+          </div>
+        </div>
+      </div>
     </div>
   </section>
 </template>
 
 <script>
     import referencesText from "@/assets/text/referencesText";
-
+    import USWDS from "uswds/src/js/components";
+    const { accordion } = USWDS;
```

Comment:
Did not know you could do this like that

Reply:
Yeah, saves us from loading in their whole package.  Just what we want, performance baby.